docs/modules/faqs.html

```diff
@@ -251,10 +274,18 @@ <h3>When loading a webpage in iOS WebView, my application gets a TypeError: unde
 						<h3>I notice that my clients lose audio and drop from the meeting if the Chrome browser on Android stays in background for several minutes, is that a known issue?</h3>
 					</a>
 					<p>This is a known issue and happens primarily due to this <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=951418">Chromium bug</a>. At present we do not have a workaround.</p>
+					<a href="#i-only-see-iphone-or-ipad-microphone-as-audio-input-in-ios-device-is-that-a-known-issue" id="i-only-see-iphone-or-ipad-microphone-as-audio-input-in-ios-device-is-that-a-known-issue" style="color: inherit; text-decoration: none;">
+						<h3>I only see iPhone or iPad microphone as audio input in iOS device, is that a known issue?</h3>
+					</a>
+					<p>This is a known issue. For more information, please refer to <a href="https://bugs.webkit.org/show_bug.cgi?id=174833">the Webkit issue</a>.</p>
 					<a href="#i-can-hear-clicking-noise-from-an-attendee-using-iphone-x-devices-x-xs-and-xr" id="i-can-hear-clicking-noise-from-an-attendee-using-iphone-x-devices-x-xs-and-xr" style="color: inherit; text-decoration: none;">
 						<h3>I can hear clicking noise from an attendee using iPhone X devices (X, XS, and XR).</h3>
 					</a>
 					<p>This is a known issue on iPhone X devices (X, XS, and XR) when using AudioContext-based APIs. For more information, see our comment in <a href="https://bugs.webkit.org/show_bug.cgi?id=204625#c6">the WebKit issue</a>.</p>
+					<a href="#i-cannot-hear-audio-from-other-attendees-using-ios-devices-once-i-unplugged-my-wired-headset" id="i-cannot-hear-audio-from-other-attendees-using-ios-devices-once-i-unplugged-my-wired-headset" style="color: inherit; text-decoration: none;">
+						<h3>I cannot hear audio from other attendees using iOS devices once I unplugged my wired headset.</h3>
+					</a>
+					<p>This is a known issue. For more information, please refer to <a href="https://bugs.webkit.org/show_bug.cgi?id=216389">the Webkit issue</a>.</p>
```

Comment:
Can we expand that 'iOS Safari has a k nown issue where it does not ...'?

Reply:
`I cannot hear audio from other attendees using iOS devices after I unplug my wired headset.`